### PR TITLE
[WIP] 3.x Migration

### PR DESCRIPTION
**Pull request by jjnesbitt**
I've followed the 2.x -> 3.x migration guide and hopefully there's no issues. The girder build succeeds and changes are seen in the front end, but as far as extensive test cases I'm not sure what to do.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -61,4 +61,9 @@
         ('database_assetstore/girder', ['plugin.json']),
     ],
     test_suite='plugin_tests',
+    entry_points={
+        'girder.plugin': [
+            'database-assetstore = database_assetstore:DatabaseAssetstorePlugin'
```
Comment (brianhelba) [inline setup.py]: This should be an underscore name.

**Comment by brianhelba** [comment]
@AlmightyYakob Girder 3 no longer provides `scripts/install_mongo.sh`, but you can fetch it from here: https://github.com/girder/girder/blob/2.x-maintenance/scripts/install_mongo.sh

You can copy that file to the local repo, then call it from there [in `.travis.yml`](https://github.com/OpenGeoscience/database_assetstore/blob/3.x_migration/.travis.yml#L43).

**Comment by jjnesbitt** [comment]
Closing the pull request for now as the plugin isn't completely working with version 3

**Comment by jjnesbitt** [comment]
The plugin seems to be working locally now, but I'm unsure as to what changes need to be done to `.travis.yml`.

**Pull request closed without merging**